src/types/profile.js

```diff
@@ -734,6 +734,17 @@ export type SampleUnits = {|
   +threadCPUDelta: ThreadCPUDeltaUnit,
 |};
 
+export type ExtraProfileInfoSection = {|
+  // section label
+  label: string,
+  entries: {|
```

Comment:
Please use `Array<>` instead of `[]`. It's not always obvious that this is an array until you see the end.